unstructured/ingest/cli/cmds/sftp.py

```diff
@@ -0,0 +1,40 @@
+import typing as t
+from dataclasses import dataclass
+
+import click
+
+from unstructured.ingest.cli.base.src import BaseSrcCmd
+from unstructured.ingest.cli.interfaces import (
+    CliConfig,
+)
+
+CMD_NAME = "sftp"
+
+
+@dataclass
+class SftpCliConfig(CliConfig):
+    username: str
+    password: str
+
+    @staticmethod
+    def get_cli_options() -> t.List[click.Option]:
```

Comment:
Would we want to support a user passing in the host/port explicitly as a cli argument?

Reply:
@rbiseck3 Would they have the choice of either as cli or in the --remote-url? I know in the url is pretty standard for sftp

Reply:
seems like once you use `urlparse` with sftp protocol, it is really expecting a hostname right after the `//` (port is optional) so maybe its natural to leave it in the --remote_url